test/dependencies.jl

```diff
@@ -33,9 +40,16 @@ end
         jdep = JSON.lower(dep)
         @test jdep == Dict("type" => "dependency", "name" => name, "uuid" => nothing, "version-major" => 0x0, "version-minor" => 0x0, "version-patch" => 0x0)
         @test dependencify(jdep) == dep
+
+        jdep_buildver = JSON.lower(dep_buildver)
+        @test jdep_buildver == Dict("type" => "dependency", "name" => name, "uuid" => nothing, "version-major" => 0x0, "version-minor" => 0x0, "version-patch" => 0x0)
+        # the build_version is currently not serialized, so the following test fails
+        @test_broken dependencify(jdep_buildver) == dep_buildver
```

Comment:
Should we make this work instead?

Reply:
I am not sure (which is why I left it as-is). I can't think of a usecase for it right now, though, and my feeling was that code I don't write also is code that won't contain bugs ;-) But perhaps @staticfloat can think of a reasons why we might want to serialize the build version?

Reply:
Ok, I don't think this is super-important now, I'm going to merge

Reply:
Hmmm, it could be important in case we ever want to e.g. know precisely which version of a JLL was pushed up.  We should probably just change whatever preprocessing we're doing to `VersionNumbers` before serializing them to include the build number.